frontend/src/container/Dashboard.jsx

```diff
@@ -157,10 +149,6 @@ export class Dashboard extends React.Component {
         }
         this.setHeadTitle()
         break
-      case 400:
-        props.history.push(PAGE.HOME)
-        props.dispatch(newFlashMessage(props.t('Unknown space')))
-        break
```

Comment:
Why delete this?

Reply:
err, never mind, I screwed this up. Fixed.